buildspec.yml

```diff
@@ -18,39 +18,7 @@ phases:
     commands:
       - echo Build started on `date`
       - echo Building the Docker image...
-      - |
-        if expr "${CODEBUILD_BUILD_ARN}" : ".*eia" >/dev/null; then
-          if expr "${CODEBUILD_BUILD_ARN}" : ".*tensorflow-1" >/dev/null; then
-            python src/main.py --buildspec $FRAMEWORK/buildspec-tf1-eia.yml --framework $FRAMEWORK
-          else
-            python src/main.py --buildspec $FRAMEWORK/buildspec-eia.yml --framework $FRAMEWORK
-          fi
-        elif expr "${CODEBUILD_BUILD_ARN}" : ".*neuron" >/dev/null; then
-          if expr "${CODEBUILD_BUILD_ARN}" : ".*tensorflow-1" >/dev/null; then
-            python src/main.py --buildspec $FRAMEWORK/buildspec-tf1-neuron.yml --framework $FRAMEWORK
-          elif expr "${CODEBUILD_BUILD_ARN}" : ".*huggingface.*" >/dev/null; then
-            PATH_TO_BULDSPEC=$(echo "$FRAMEWORK" | tr _ /)
-            python src/main.py --buildspec $PATH_TO_BULDSPEC/buildspec-neuron.yml --framework $FRAMEWORK
-          else
-            python src/main.py --buildspec $FRAMEWORK/buildspec-neuron.yml --framework $FRAMEWORK
-          fi
-        elif expr "${CODEBUILD_BUILD_ARN}" : ".*graviton" >/dev/null; then
-          python src/main.py --buildspec $FRAMEWORK/buildspec-graviton.yml --framework $FRAMEWORK
-        else
-          if expr "${CODEBUILD_BUILD_ARN}" : ".*tensorflow-1" >/dev/null; then
-            python src/main.py --buildspec $FRAMEWORK/buildspec-tf1.yml --framework $FRAMEWORK
-          elif expr "${CODEBUILD_BUILD_ARN}" : ".*huggingface.*" >/dev/null; then
-            # In HuggingFace case FRAMEWORK looks like "huggingface_<framework_name>"
-            # converting "huggingface_<framework_name>" into huggingface/<framework_name>
-            # to create a correct path to buildspec files
-            PATH_TO_BULDSPEC=$(echo "$FRAMEWORK" | tr _ /)
-            python src/main.py --buildspec $PATH_TO_BULDSPEC/buildspec.yml --framework $FRAMEWORK
-          elif expr "${CODEBUILD_BUILD_ARN}" : ".*habana.*" >/dev/null; then
-            python src/main.py --buildspec habana/$FRAMEWORK/buildspec.yml --framework $FRAMEWORK
-          else
-            python src/main.py --buildspec $FRAMEWORK/buildspec.yml --framework $FRAMEWORK
-          fi
-        fi
+      - python src/main.py --buildspec $FRAMEWORK_BUILDSPEC_FILE --framework $FRAMEWORK
```

Comment:
FRAMEWORK_BUILDSPEC_FILE is a path I assume?

Reply:
Yeah, it is the file path.